src/test/java/nextstep/subway/line/LineAcceptanceTest.java

```diff
@@ -1,72 +1,165 @@
+import org.springframework.http.MediaType;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.http.HttpStatus.*;
 
 @DisplayName("지하철 노선 관련 기능")
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
 public class LineAcceptanceTest extends AcceptanceTest {
+
+    private static final String PATH = "/lines";
+
     @DisplayName("지하철 노선을 생성한다.")
     @Test
     void createLine() {
         // when
-        // 지하철_노선_생성_요청
+        final ExtractableResponse<Response> 신분당선 = 지하철_노선_생성요청("신분당선","red");
 
         // then
-        // 지하철_노선_생성됨
+        응답코드_확인(신분당선, CREATED);
     }
 
     @DisplayName("지하철 노선 목록을 조회한다.")
     @Test
     void getLines() {
         // given
-        // 지하철_노선_등록되어_있음
-        // 지하철_노선_등록되어_있음
+        final ExtractableResponse<Response> 경강선 = 지하철_노선_생성요청("경강선", "blue");
+        final ExtractableResponse<Response> 신분당선 = 지하철_노선_생성요청("신분당선", "red");
 
         // when
-        // 지하철_노선_목록_조회_요청
+        final ExtractableResponse<Response> response = 지하철_노선_목록조회요청();
 
         // then
-        // 지하철_노선_목록_응답됨
-        // 지하철_노선_목록_포함됨
+        응답코드_확인(response, OK);
+        assertThat(getLineIds(response)).containsAll(extractLineIds(경강선, 신분당선));
     }
 
     @DisplayName("지하철 노선을 조회한다.")
     @Test
     void getLine() {
         // given
-        // 지하철_노선_등록되어_있음
+        final ExtractableResponse<Response> 경강선 = 지하철_노선_생성요청("경강성", "blue");
 
         // when
-        // 지하철_노선_조회_요청
+        final ExtractableResponse<Response> response = 지하철_노선_조회요청("/lines/1");
 
         // then
-        // 지하철_노선_응답됨
+        응답코드_확인(response, OK);
+        assertThat(getLineId(response)).isEqualTo(getLineId(경강선));
     }
 
+
     @DisplayName("지하철 노선을 수정한다.")
     @Test
     void updateLine() {
         // given
-        // 지하철_노선_등록되어_있음
+        지하철_노선_생성요청("경강성", "blue");
+        final LineRequest 신분당선 = new LineRequest("신분당선", "red");
 
         // when
-        // 지하철_노선_수정_요청
+        final ExtractableResponse<Response> response = 지하철_노선_수정요청(신분당선);
 
         // then
-        // 지하철_노선_수정됨
+        응답코드_확인(response, OK);
+        assertThat(getLineName(response)).isEqualTo(신분당선.getName());
     }
 
+
     @DisplayName("지하철 노선을 제거한다.")
     @Test
     void deleteLine() {
         // given
-        // 지하철_노선_등록되어_있음
+        지하철_노선_생성요청("경강성", "blue");
 
         // when
-        // 지하철_노선_제거_요청
+        final ExtractableResponse<Response> response = 지하철_노선_삭제요청();
 
         // then
-        // 지하철_노선_삭제됨
+        응답코드_확인(response, NO_CONTENT);
+    }
+
+    private String getLineName(ExtractableResponse<Response> response) {
+        return response.body().jsonPath().getString("name");
+    }
+
+    private long getLineId(ExtractableResponse<Response> response) {
+        return response.body().jsonPath().getLong("id");
```

Comment:
```suggestion
        return response.as(LineResponse.class).getId()
```
getId를 활용하여 값을 조회할 수 도 있어요~

Reply:
RestAssured 공식문서를 자세히 봐야겠네요. 감사합니다